tools/fastq_provider/fastq_provider.xml

```diff
@@ -1,78 +1,160 @@
-<tool id="fastq_provider" name="Atlas FASTQ provider" version="@TOOL_VERSION@+galaxy" python_template_version="3.5" profile="20.01">
+<tool id="fastq_provider" name="Atlas FASTQ provider" version="@TOOL_VERSION@+galaxy1" python_template_version="3.5" profile="20.01">
```

Comment:
```suggestion
<tool id="fastq_provider" name="Atlas FASTQ provider" version="@TOOL_VERSION@+galaxy0" python_template_version="3.5" profile="20.01">
```

Reply:
Done in d479636578ecba03d14b6dd0ad97fd2222019e8c

Reply:
Resolving this conversation as its done in d479636578ecba03d14b6dd0ad97fd2222019e8c